Python/Product/Analysis/Interpreter/NoPackageManager.cs

```diff
@@ -80,5 +90,9 @@ public sealed class NoSuppression : IDisposable {
         public Task<PackageSpec> GetInstallablePackageAsync(PackageSpec package, CancellationToken cancellationToken) {
             return Task.FromResult(new PackageSpec());
         }
+
+        public string GetInstallCommandDisplayName(string searchQuery) {
+            throw new NotImplementedException();
```

Comment:
Don't like throwing `NotImplementedException` from real code. This doesn't seem like a good candidate for `NotSupportedException`, so probably should just return an empty string.